### ISSUE TITLE
Invalid aria-description attribute.

### ISSUE THREAD

**Issue by bhanu345-cmd** [ag-grid/ag-grid#6945]
Accessibility fast pass text got failed at headers of ag grid because of aria-description attribute.

React version: ^18.0.0
Ag grid version: ^30.0.0

Using Accessibility insights for web for testing accessibility issues.

<img width="313" alt="image" src="https://github.com/ag-grid/ag-grid/assets/63151401/58ec640c-48a9-4828-b05a-26929edbde29">

**Comment by zoheil-ag** [comment]
Hi @bhanu345-cmd,

Thank you for reporting this.

We'd be happy to look into this, but the most efficient way for us to do so would be for you to provide a live example reproducing the issue.

Please send us a live plunker sample which shows the issue - which part of the grid is this referring to? You can use one of the examples from our website as a starting point. Do let us know if you need further guidance on this.

We're looking forward to your response.

Kind regards,
Zoheil

**Comment by bhanu345-cmd** [comment]
Code sandbox link : https://codesandbox.io/s/unruffled-bohr-w7fqj8?file=/src/App.tsx

Reproduce steps:
1) Open code sand box browser in new window.
2) Run fast pass using Accessibility insights for web.
you will observe below mentioned 

<img width="924" alt="image" src="https://github.com/ag-grid/ag-grid/assets/63151401/75e2a2b0-c058-4092-9193-6e2b6d34661a">

**Comment by zoheil-ag** [comment]
Thank you very much for the extra information @bhanu345-cmd.

Currently there's no way to implement this behavior with AG Grid. However, we agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference and description:
AG-9085 [Accessibility] Allow ARIA attributes for column headers to be named correctly 

We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. Please use the functionality currently provided by AG Grid until this is delivered at a later point.

See whether this item will be in the next release by checking the NEXT RELEASE checkbox on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil
